test/excludes/LeftOuterJoinAssociationTest.rb

```diff
@@ -0,0 +1,2 @@
+exclude :test_construct_finder_sql_applies_aliases_tables_on_association_conditions, "The test fails because the query result order is not guaranteed."
```

Comment:
This won't stop the version you wrote about from running, right?

Reply:
It'll prevent the ActiveRecord version of this test from running, but it won't stop `test/cases/associations/left_outer_join_association_test.rb ` from running. Does that answer your question?

Reply:
Yes, thanks!